Reimport css/css-scrollbars WPT tests.

lukewarlow opened the pull request:
#### 12222b2a21138d94248c76a0f49a0c09f41291f1
<pre>
Reimport css/css-scrollbars WPT tests.
<a href="https://bugs.webkit.org/show_bug.cgi?id=257570">https://bugs.webkit.org/show_bug.cgi?id=257570</a>

Reviewed by Tim Nguyen.

Reimport css/css-scrollbars WPT tests, this is to get the scrollbar-color-parsing test.

WPT Commit: ceb8a99f802f1b8193ce8a611374c0991933334b

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/input-scrollbar-color-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/input-scrollbar-color-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/input-scrollbar-color.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-color-parsing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-color-parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/w3c-import.log:

Canonical link: <a href="https://commits.webkit.org/264793@main">https://commits.webkit.org/264793@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/00670d32c07ad1b2807e781a6f2703b53d3844f3

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/8629 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/8919 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/9134 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/10285 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/8646 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/10908 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/8892 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/11508 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/8775 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/9795 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/7731 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/10443 "Built successfully") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/7095 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/7888 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/15418 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [  ~~🧪 api-ios~~](https://ews-build.webkit.org/#/builders/13/builds/8214 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/8036 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/11379 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/8523 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/17/builds/6949 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/7793 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/7799 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/2098 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/12004 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/8264 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/resources/import-expectations.json:
```diff
@@ -97,6 +97,7 @@
     "web-platform-tests/cors": "import",
     "web-platform-tests/credential-management": "import",
     "web-platform-tests/css": "skip",
+    "web-platform-tests/css-scrollbars": "import",
```
Can you undo this change? It’s redundant, web-platform-tests/css/css-scrollbars is below

lukewarlow committed:
Reimport css/css-scrollbars WPT tests.
https://bugs.webkit.org/show_bug.cgi?id=257570

Reviewed by Tim Nguyen.

Reimport css/css-scrollbars WPT tests, this is to get the scrollbar-color-parsing test.

WPT Commit: ceb8a99f802f1b8193ce8a611374c0991933334b

* LayoutTests/imported/w3c/resources/resource-files.json:
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/input-scrollbar-color-expected.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/input-scrollbar-color-ref.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/input-scrollbar-color.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-color-parsing-expected.txt: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/scrollbar-color-parsing.html: Added.
* LayoutTests/imported/w3c/web-platform-tests/css/css-scrollbars/w3c-import.log:

Canonical link: https://commits.webkit.org/264793@main
